apps/prairielearn/elements/pl-dropdown/pl-dropdown.mustache

```diff
@@ -1,6 +1,11 @@
 {{#question}}
     <span class="d-inline-block mb-3">
-        <select name="{{answers-name}}" class="form-select w-auto" {{^editable}}disabled{{/editable}}>
+        <select
+            name="{{answers-name}}"
+            class="form-select w-auto"
+            {{^editable}}disabled{{/editable}}
+            {{#aria-labelledby}}aria-labelledby="{{aria-labelledby}}"{{/aria-labelledby}}
```

Comment:
Two notes:

- `pl-dropdown` is deprecated, there's little value in further maintenance.
- If you really do want to change things here, a `label` attribute that's used as `aria-label` is probably a better API. This mirrors other elements. Consider a default like `Multiple choice options` to mirror what `pl-multiple-choice` does. Note that in `pl-multiple-choice`, the label isn't even configurable; it may not need to be here either.

Reply:
The reason I did this is based on usage patterns, there is an element directly above the dropdown that contains the text. So then you can avoid duplicating that text. I'm fine changing it though.

Reply:
Changed in [785357d](https://github.com/PrairieLearn/PrairieLearn/pull/11844/commits/785357da395b470176b257d2ad9cac4482f141ef).